src/features/locators/components/LocatorEditDialog.tsx

```diff
@@ -23,27 +25,33 @@ import FormItem from "antd/es/form/FormItem";
 import { LocatorType } from "../../../common/types/locatorType";
 import { getLocator, getXPathByPriority } from "../utils/locatorOutput";
 
+import { sendMessage } from "../../../pageServices/connector";
+import { evaluateXpath } from "../utils/utils";
+
+import { generateId, getElementFullXpath } from "../../../common/utils/helpers";
+
 interface Props extends Locator {
```

Comment:
we need to make Locator props optional or pass an empty locator, according to this interface

Reply:
fixed